x-pack/plugins/actions/server/lib/get_token_count_from_invoke_stream.ts

```diff
@@ -170,8 +137,10 @@ function getMessageLength(buffer: Uint8Array): number {
   return view.getUint32(0, false);
 }
 
-const parseOpenAIResponse = (responseBody: string) =>
-  responseBody
+const parseOpenAIChunks = (responseBodyChunks: string[]) =>
+  responseBodyChunks
+    .map((chunk) => chunk.toString())
```

Comment:
We need to call `toString()` on openai chunks. The type `responseBodyChunks: string[]` is therefore incorrect